dspace/config/registries/openaire4-types.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  This document was based on the OpenAIRE4 Guidelines for Literature Repositories
+  https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/index.html
+  -->
+  
+<dspace-dc-types>
+    <dspace-header>
+        <title>OpenAIRE4 fields definition</title>
```

Comment:
This is where I feel like we're starting to get into a gray area with this backport.  I'm honestly not sure I like adding new metadata schemas & fields in minor releases.  I know it shouldn't be a big deal overall, but it's not a great practice (especially since 7.x isn't out the door yet, and there's always a chance these fields could change slightly still).

Reply:
are the extra registries automatically loaded during update? if so, I agree with @tdonohue  to remove them and just add a mention about them in the documentation. If they are not loaded automatically I'm personally ok with keeping them here again just adding a small mention in the documentation

Reply:
Now this file is not listed in the Flayway callback DatabaseRegistry. So, at moment this metadata is not loaded automatically.

Reply:
@abollini : Good question. During any install/upgrade, only registries listed here are automatically loaded: https://github.com/DSpace/DSpace/blob/dspace-6_x/dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseRegistryUpdater.java#L70-L76

(As a sidenote, as of 7.x, this list of registries is configurable in the `registry.metadata.load` configuration).

So, in this situation, these new schemas/types would *not* be loaded automatically during an upgrade.   However, I'm still slightly concerned here about encouraging sites to use these metadata schemas/fields *before* the Configurable Entities feature exists.  It seems like it'd require us to document a process for migrating old DSpace 6 Items with these metadata fields into new DSpace 7 Entities.  If others feel strongly that this is a good idea, we can backport this (since they won't be automatically loaded by default), but it just seems like an area of the backport that isn't strictly necessary & could complicate the upgrade to DSpace 7.

Reply:
Since these are not listed in Flyway DatabaseRegistry by default, these are OK.  However, OpenAIRE v4 documentation would need to be written to describe how to enable these.